src/annotator/guest.js

```diff
@@ -577,7 +587,8 @@ export default class Guest {
    * Remove the anchors and associated highlights for an annotation from the document.
    *
    * @param {string} tag
-   * @param {boolean} [notify] - For internal use. Whether to emit an `anchorsChanged` notification
+   * @param {boolean} [notify] - For internal use. Whether to inform the `host`
```

Comment:
```suggestion
   * @param {boolean} [notify] - For internal use. Whether to inform the host
```

"host frame" is a concept rather than an identifier. If you are introducing a concept in documentation, you can use italics for that. In this case though I think we can assume that the concept of a host vs guest is explained elsewhere.